format/FormatNexusEigerDLS16M.py

```diff
@@ -60,6 +69,11 @@ def __init__(self, image_file, **kwargs):
 
         super(FormatNexusEigerDLS16M, self).__init__(image_file, **kwargs)
         self._dynamic_shadowing = self.has_dynamic_shadowing(**kwargs)
+        try:
+            meta = image_file.replace("_master.h5", "_meta.h5")
+            self._bit_depth_image = get_bit_depth_from_meta(meta)
+        except Exception:
```

Comment:
Is it possible to be a bit more specific about what type of exception we expect here?

Reply:
Could be `KeyError` or `OSError` or `FileNotFound` or ... but would not change anything right? i.e. if something goes wrong, ignore...